conf/modules.config

```diff
@@ -1093,6 +1093,15 @@ process{
         ext.prefix       = {"${meta.id}.somatic_snvs"}
     }
 
+    //TIDDIT
+    withName: 'NFCORE_SAREK:SAREK:PAIR_VARIANT_CALLING:RUN_TIDDIT_NORMAL:TABIX_BGZIP_TIDDIT_SV' {
+        ext.prefix       = {"${meta.id}.normal.vcf"}
+    }
+
+    withName: 'NFCORE_SAREK:SAREK:PAIR_VARIANT_CALLING:RUN_TIDDIT_TUMOR:TABIX_BGZIP_TIDDIT_SV' {
+        ext.prefix       = {"${meta.id}.tumor.vcf"}
+    }
+
```

Comment:
We would also like to publish/save the svdb output.

Reply:
done, pls check that my command is not missing anything

Reply:
hm, something seems to be missing indeed, the output is not published